src/components/PigeanTable.vue

```diff
@@ -19,19 +19,19 @@
             :sort-desc="true"
           >
             <template #cell(gene)="r">
-              <a :href="`/pigean/gene.html?gene=${r.item.gene}`">
+              <a :href="`/pigean/gene.html?gene=${r.item.gene}&sigma=sigma${sigma}`">
                 {{ r.item.gene }}
               </a>
             </template>
             <template #cell(phenotype)="r">
               <a v-if="!!phenotypeMap[r.item.phenotype]"
-                :href="`/pigean/phenotype.html?phenotype=${r.item.phenotype}`">
+                :href="`/pigean/phenotype.html?phenotype=${r.item.phenotype}&sigma=sigma${sigma}`">
```

Comment:
sigma= should be enough, then you don't have to use slice in computed, and just use param directly

Reply:
This is intentional. Sigma needs to be a string in the key params and a number in the query. This is because 0 is a valid sigma, but 0 as a key param is taken as the equivalent of FALSE.

Reply:
In that case, you can also just import and use the query-string library [directly](https://github.com/broadinstitute/dig-dug-portal/blob/b606c4e6d808a134af2f5ce855226ddc2904ee96/src/utils/keyParams.js#L20-L23). This way, you can make sure it behaves how you want it to, boolean, number or string.

Reply:
I didn't realize we had that feature! That's great, thank you.